autoblocks/_impl/prompts/models.py

```diff
@@ -80,7 +80,7 @@ def str_version(self) -> str:
 
 
 class PromptMinorVersion(FrozenModel):
-    version: Union[Enum, List[WeightedMinorVersion]]
+    version: Union[Enum, List[WeightedMinorVersion]]  # type: ignore
```

Comment:
need to do more research here, `WeightedMinorVersion` requires generics but we don't care what it is in this case, just that it's an instance of `WeightedMinorVersion`. tried wrapping in `Type` or using `Any` for the generics but couldn't get it working